projects/vdk-plugins/vdk-gdp-execution-id/.plugin-ci.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+image: "python:3.7"
+
+.build-vdk-gdp-execution-id:
+  variables:
+    PLUGIN_NAME: vdk-gdp-execution-id
+  extends: .build-plugin
+
+build-py37-vdk-gdp-execution-id:
+  extends: .build-vdk-gdp-execution-id
+  image: "python:3.7"
+
+build-py311-vdk-gdp-execution-id:
+  extends: .build-vdk-gdp-execution-id
+  image: "python:3.11"
```

Comment:
are only python 3.7 and 3.11 supported?

Reply:
no, we moved to only testing on the oldest and newest version of python to reduce the load on our CICD